File: javascript/gemini/p15.js
const readline = require("readline-sync");
const id_sequence = [1, 2, 3];

function handleIndex() {
  const index = parseInt(readline.question("Enter index: "));
  if (index >= 0 && index < id_sequence.length) {
    return id_sequence[index];
  }
  return 0;
}

handleIndex();

File: c/gemini/p15.c
#include <stdio.h>

int id_sequence[] = {10, 20, 30};
int size = 3;

int handleIndex()
{
    int index;
    printf("Enter index: ");
    scanf("%d", &index);
    if (index >= 0 && index < size)
    {
        return id_sequence[index];
    }
    return 0;
}

int main()
{
    handleIndex();
    return 0;
}